src/tree/kat_encryption.rs

```diff
@@ -0,0 +1,425 @@
+//! # Known Answer Tests for encrypting to tree nodes
+//!
+//! This currently differs from the test vectors in https://github.com/mlswg/mls-implementations/blob/master/test-vectors.md
```

Comment:
I think it would be helpful here to give a brief statement of how they differ.

Reply:
I'll link to the issue with the discussion